helpers/helpers.go

```diff
@@ -0,0 +1,47 @@
+package helpers
```

Comment:
This random string generator seems too complex for this package. I think it should be a separate `helpers/rand` package after all.